sdks/java/core/src/main/java/org/apache/beam/sdk/coders/StructuredCoder.java

```diff
@@ -98,6 +101,34 @@ public String toString() {
     return builder.toString();
   }
 
+  public void encode(T value, OutputStream outStream)
```

Comment:
Implemented the base encode/decode methods (though this opens us up to infinite recursion and no abstract methods, it should be short term). However, it's unsafe to unconditionally delegate the Outer ones to nested until everything is migrated.